### PR TITLE
[UG] YJ part + command summary changes 

### PR DESCRIPTION
**Pull request by Yonggiee**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -249,40 +245,32 @@ image::ug-findcExamplePart2.png[]
 image::ug-findcExamplePart3.png[]
 
 ==== Common errors / problems
-You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them. You will also get a better understanding of the reply from the result box when using `find-c`.
+You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them.
```
Comment (AaronCQL) [inline docs/UserGuide.adoc]: I think we can afford to be more concise? How about "here are some common errors and problems:"

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -249,40 +245,32 @@ image::ug-findcExamplePart2.png[]
...
-You might wonder why `find-c` do not show the clients even though the client is clearly in your client list records. It might be possible that you have entered an incomplete name and does not match any word in the name of that client. The example below might help you understand better.
+You might wonder why `find-c` do not show the clients even though the client is clearly in your client list records. It might be possible that you have entered an incomplete name and does not match any word in the name of that client.
 
-Example:
 You want to find the client named "Bryan Low" in *Client List*. Below shows that "Bryan Low" indeed exists and is recorded.
 
 image::ug-findcIncompleteNamePart1.png[]
 
-Type `find-c bry` into *Command Box*, and press `Enter` to execute it.
+Enter `find-c bry` into *Command Box*. You will obtain "0 clients listed!" in *Result Box*.
```
Comment (AaronCQL) [inline docs/UserGuide.adoc]: Might want to put the **Result Box** in the beginning: "**Result Box** shows `0 clients listed!`"

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -763,54 +751,47 @@ To correct this, you have to enter a valid `INDEX` that is not more than the num
 
 *Format*: `add-e n/EXERCISE_NAME d/DATE [ew/EXERCISE_WEIGHT] [reps/REPS] [sets/SETS]`
 
+[NOTE]
+This command can only be used when you have a client in view; make sure you know how to view a client first. Refer to <<view-c-command>> for more information.
```
Comment (AaronCQL) [inline docs/UserGuide.adoc:755]: I kinda rephrased this slightly for my version, can just KIV first.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -819,43 +800,32 @@ Reps: 4 +
...
 
-. View the client that you want to add the exercise to. For information on how to view the client, you can refer to the `view-c` section. For this example, we will add the exercise to the first client in the list, `Alex Yeoh`. After you entered `view-c 1`, you will see `Alex Yeoh` detailed information and a table of his recorded exercises.
+. Use the `view-c` command to view "Alex Yeoh".
++
+
+. Type the information of the exercise, matching each detail of the exercise to the parameter. In this case, type `add-e n/Bench press d/12-02-2020 reps/4 ew/100 sets/4` into *Command Box*. Press enter to execute it.
 +
 image::ug-addeExamplePart1.png[]
-. Now type the information of the exercise, matching each detail of the exercise to the parameter. In this case, you will have to type `add-e n/Bench press d/12-02-2020 reps/4 ew/100 sets/4` into *Command Box*. Press enter to execute it. To recap,
+. After it has been successfully entered, *Result Box* will display that exercise has been sucessfully recorded. You will also be able to see the recorded exercise in the *Exercise Table*.
```
Comment (AaronCQL) [inline docs/UserGuide.adoc]: > After it has been successfully entered

After the command has been successfully executed

> **Result Box** will display that exercise has been sucessfully recorded

**Result Box** will notify that the exercise has been recorded

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -1375,6 +1336,7 @@ Examples:
 |=======================================================================
 
 === Exercise Commands
+The table below shows the commands that manipulate the details of the exercises done by the client.
```
Comment (AaronCQL) [inline docs/UserGuide.adoc]: "done by **a** client"

**Review by Dban1 — APPROVED** [review]
Looks good so far. Let me merge first

**Pull request merged**